### DISCUSSION TITLE
Binaries for Sciter Lite on Linux

### DISCUSSION THREAD

**Discussion by niklasbuesing** [c-smile/sciter-js-sdk#35] [Q&A] [answered]
Hi,

First of all, thank you for your efforts, having JS support in Sciter will greatly help with IDE support.
It's probably not a high priority right now, but is there an ETA for the release of SciLite Linux binaries?

Thanks!

**Comment by c-smile** [comment, marked as answer]
Sciter.Lite for Linux is coming too.
 
> having JS support in Sciter will greatly help with IDE support.

Not sure I understand this statement. What IDE are you talking about?

**Reply by niklasbuesing** [reply]
Thanks.

I'm using IntelliJ, with JS stuff like syntax highlighting, auto-complete and other IDE features will work properly.
I found some syntax definitions for tiscript but they all fall apart quickly when you're using JSX for example.
